superdesk/default_settings.py

```diff
@@ -116,6 +116,9 @@ def local_to_utc_hour(hour):
 #: full mongodb connection uri, overrides ``MONGO_DBNAME`` if set
 MONGO_URI = env('MONGO_URI', 'mongodb://localhost/%s' % MONGO_DBNAME)
 
+#: legal archive on switch, set to ``0`` to disable it
+LEGAL_ARCHIVE = env('SUPERDESK_LEGAL_ARCHIVE', '1') != '0'
```

Comment:
I think we should be consistent with config values, probably just empty string is enough for disabling.

Also I think env variable should be `LEGAL_ARCHIVE` instead of `SUPERDESK_LEGAL_ARCHIVE`, because we don't have `SUPERDESK_` prefix for setting like 
```
root@sds-naspeh:/opt/superdesk/server-core# env | grep LEGAL
LEGAL_ARCHIVE_URI=mongodb://data-sd/sds-naspeh_la
```
and many others.